test/robot/build/package.go

```diff
@@ -173,11 +182,25 @@ func (pkg *Package) mergeTool(ctx context.Context, tool *ToolSet) {
 }
 
 // GetTools will return the toolset that match the abi, if there is one.
-func (p *Package) GetTools(abi *device.ABI) *ToolSet {
+func (p *Package) GetHostTools(abi *device.ABI) *ToolSet {
 	for _, t := range p.Tool {
 		if t.Abi.SameAs(abi) {
 			return t
 		}
 	}
 	return nil
 }
+
+func (p *Package) GetTargetTools(hostAbi *device.ABI, targetAbi *device.ABI) *AndroidToolSet {
```

Comment:
`// GetTargetTools will return the target toolset that matches the host and target abi, if there is one.`

Would have been caught with VSCode and the go plugin.